repository/repository.js

```diff
@@ -0,0 +1,50 @@
+export class Repository {
+  constructor(Document) {
+    this.Document = Document;
+  }
+  async getItems() {
+    try {
+      const items = await this.Document.find();
+
+      return items;
+    } catch (error) {
+      return error.message;
+    }
+  }
+  async createItem(item) {
+    const newItem = new this.Document(item);
+    try {
+      await newItem.save();
+      console.log(newItem);
+      return newItem;
+    } catch (error) {
+      return error.message;
+    }
+  }
+  async getItemById(id) {
+    let item;
+    try {
+      item = await this.Document.findById(id);
+    } catch (error) {a.message("error occured when looking for item");
+    }
+    if (!item) {
+      return `Cannot find item of id: ${id}`;
```

Comment:
I mentioned it already, when the item with a given ID is not found then return null, not a string with a message.